mesmer/xarray_utils/grid.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import xarray as xr
+from packaging.version import Version
+
+
+def to_unstructured(
+    obj, *, x_dim="lon", y_dim="lat", cell_dim="cell", multiindex=False, dropna=True
+):
+    """stack a regular grid to an unstructured grid
```

Comment:
```suggestion
    """Stack a lat-lon grid to a 1D grid
```

In line with comments about calling this 'stack' rather than 'to_unstructured' in an earlier comment